tests/hare/config.yaml

```diff
@@ -27,10 +27,10 @@ bootstrap:
     hare-round-duration-sec: '10'
     hare-wakeup-delta: '5'
     layer-duration-sec: '80'
-
+    executable-path: '/bin/go-hare'
 client:
-  image: 'spacemeshos/go-spacemesh:develop'
-  command: '/bin/go-hare'
+  image: 'spacemeshos/go-spacemesh-dev:harness'
```

Comment:
dont push these changes. image should stay develop

Reply:
fixed in 704b0c0b747b960ba1d5920e85d81182eaa660a8